calico_node/filesystem/etc/calico/confd/templates/tunl-ip.template

```diff
@@ -0,0 +1,7 @@
+We must dump all pool data to this file to trigger a resync.
+Otherwise, confd notices the file hasn't changed and won't
+run our python update script.
+ 
+{{range ls "/pool"}}{{$data := json (getv (printf "/pool/%s" .))}}
```

Comment:
Worth only writing out the IPIP pools - that way we only do an update when an IPIP pool is added or deleted.